src/test/readaheadmanager_test.cpp

```diff
@@ -24,10 +24,10 @@ class StubReader : public CachingReader {
     }
 };
 
-class StubLoopControl : public EngineControl {
+class StubLoopControl : public LoopingControl {
```

Comment:
for this to be safe, please override all EngineControl functions that LoopingControl also overrides.  That way if something should change eventually and RAMAN calls one of those functions, we're guaranteed to get the Stub version.  They can be noops for now or even assert(false) calls if they are unused.